include/lsst/afw/math/Interpolate.h

```diff
@@ -24,6 +24,7 @@
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
 #include "lsst/base.h"
+#include "ndarray_fwd.h"
```

Comment:
Looks to me like `ndarray_fwd.h` is a top level include (look in `$NDARRAY_DIR/include`)